src/main/resources/templates/macros/json/embeddings.vm

```diff
@@ -54,7 +56,26 @@
 </div>
 #end
 
-
+#macro(includeVideoEmbedding $embedding, $format ,$index)
+  #set($index = $index + 1)
+  <div class="embedding indention">
+    #set($embeddingId = $counter.next())
+    <div data-toggle="collapse" data-target="#embedding-$embeddingId" class="collapsable-control">
+      <a>#if($embedding.getName())$embedding.getName()#{else}Attachment $index ($format)#end</a>
+      <a href="embeddings/$embedding.getFileName()" download target="_blank">
+        <span class="download-button glyphicon glyphicon-download-alt"></span>
+      </a>
+    </div>
+    <div id="embedding-$embeddingId" class="collapse collapsable-details">
+      <div class="embedding-content">
+        <video width="854" height="480" controls>
+          <source src="embeddings/$embedding.getFileName()" type="video/mp4">
```

Comment:
I will accept this PR when you add sample video to sample.json so it can be presented in live demo http://damianszczepanik.github.io/cucumber-html-reports/overview-features.html

It can be anything, ideally small. Make sure you meet license once this video becomes part of this project

Reply:
There is already a Base64 data with 1-second video. 
[Original-link](https://youtu.be/Wch3gJG2GJ4)

If this is not acceptable, I will make own video.

Reply:
can't see info about license - recommend to prepare own 'hello world' or similar

will be nice if you add also integration test for video but it is not mandatory

Reply:
I've added a new data sample in `sample.json` with my own video.